### PR TITLE
Try adding a zuul test

### PR DESCRIPTION
**Pull request by jamielennox**
Signed-off-by: Jamie Lennox <jamielennox@gmail.com>

### CODE REVIEW

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by anne-bonny** [comment]
Build failed.

- https://server/job/tox-py27/0/ : POST_FAILURE in 1m 20s
- https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 23s

**Comment by anne-bonny** [comment]
Build failed.

- https://server/job/tox-py27/0/ : POST_FAILURE in 1m 22s
- https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 23s

**Comment by anne-bonny** [comment]
Build failed.

- https://server/job/tox-py27/0/ : POST_FAILURE in 1m 22s
- https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 24s

**Comment by anne-bonny** [comment]
Build failed.

- https://server/job/tox-py27/0/ : POST_FAILURE in 1m 22s
- https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 24s

**Comment by anne-bonny** [comment]
Build failed.

- https://server/job/tox-py27/0/ : POST_FAILURE in 1m 52s
- https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 54s

**Comment by anne-bonny** [comment]
Build failed.

- https://server/job/tox-py27/0/ : POST_FAILURE in 1m 52s
- https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 53s

**Comment by anne-bonny** [comment]
Build failed.

- https://server/job/tox-py27/0/ : POST_FAILURE in 1m 39s
- https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 39s

**Comment by anne-bonny** [comment]
Build failed.

- https://server/job/tox-py27/0/ : POST_FAILURE in 1m 38s
- https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 38s

**Comment by anne-bonny** [comment]
Build failed.

- https://server/job/tox-py27/0/ : POST_FAILURE in 1m 54s
- https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 41s

**Comment by anne-bonny** [comment]
Build failed.

- https://server/job/tox-py27/0/ : POST_FAILURE in 2m 00s
- https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 41s

**Comment by anne-bonny** [comment]
Build failed.

- https://server/job/tox-py27/0/ : POST_FAILURE in 2m 05s
- https://server/job/tox-pep8/0/ : POST_FAILURE in 2m 08s

**Comment by anne-bonny** [comment]
Build failed.

- https://server/job/tox-py27/0/ : POST_FAILURE in 1m 52s
- https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 54s

**Comment by anne-bonny** [comment]
Build failed.

- https://server/job/tox-py27/0/ : POST_FAILURE in 1m 52s
- https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 54s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 https://server/job/tox-py27/0/ : POST_FAILURE in 2m 06s
- tox-pep8 https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 46s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 https://server/job/tox-py27/0/ : POST_FAILURE in 1m 53s
- tox-pep8 https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 54s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 https://server/job/tox-py27/0/ : POST_FAILURE in 2m 05s
- tox-pep8 https://server/job/tox-pep8/0/ : POST_FAILURE in 2m 06s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 https://server/job/tox-py27/0/ : POST_FAILURE in 1m 40s
- tox-pep8 https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 54s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 https://server/job/tox-py27/0/ : POST_FAILURE in 1m 40s
- tox-pep8 https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 54s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 https://server/job/tox-py27/0/ : POST_FAILURE in 1m 53s
- tox-pep8 https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 41s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 https://server/job/tox-py27/0/ : POST_FAILURE in 1m 41s
- tox-pep8 https://server/job/tox-pep8/0/ : POST_FAILURE in 1m 52s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 finger://zuul.v3.opentechsjc.bonnyci.org/d848c1dd9c8a4d15934d31b179ef7220 : POST_FAILURE in 1m 11s
- tox-pep8 finger://zuul.v3.opentechsjc.bonnyci.org/e29bc9aeec8442ceaab029f5b0cfc402 : POST_FAILURE in 1m 24s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 finger://zuul.v3.opentechsjc.bonnyci.org/83f6a5a9cae84f6f891d2a03a5e9ecb7 : POST_FAILURE in 1m 35s
- tox-pep8 finger://zuul.v3.opentechsjc.bonnyci.org/cc24e8a76d3d4873a6a2f3c4be50a4be : POST_FAILURE in 1m 11s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 finger://zuul.v3.opentechsjc.bonnyci.org/e32933f27dea4cafbc22da03bafac380 : POST_FAILURE in 1m 08s
- tox-pep8 finger://zuul.v3.opentechsjc.bonnyci.org/e9841e9ae17f4f4f8ccc88921cd7afcc : POST_FAILURE in 1m 08s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 finger://zuul.v3.opentechsjc.bonnyci.org/4dd7ea1add53411495269e97396eb227 : POST_FAILURE in 31m 21s
- tox-pep8 finger://zuul.v3.opentechsjc.bonnyci.org/af2ce51d63104a778950e3ca40be5079 : POST_FAILURE in 1m 23s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 finger://zuul.v3.opentechsjc.bonnyci.org/da8a8539280c4d4ea8d97af0648576e6 : POST_FAILURE in 1m 11s
- tox-pep8 finger://zuul.v3.opentechsjc.bonnyci.org/6aa96ff3c41843b7aef90f0cc76faccd : POST_FAILURE in 1m 11s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 finger://zuul.v3.opentechsjc.bonnyci.org/a29bd4910a6743db9043b1c5b0d9b28d : POST_FAILURE in 1m 21s
- tox-pep8 finger://zuul.v3.opentechsjc.bonnyci.org/777d33ca58d747fbac474b3f3060105f : POST_FAILURE in 1m 20s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 finger://zuul.v3.opentechsjc.bonnyci.org/307455b445524c5a8419d4587fb670aa : POST_FAILURE in 1m 11s
- tox-pep8 finger://zuul.v3.opentechsjc.bonnyci.org/2477a0d65e2e48debe515d9644289dcc : POST_FAILURE in 1m 24s

**Comment by anne-bonny** [comment]
Build failed.

- tox-py27 finger://zuul.v3.opentechsjc.bonnyci.org/ecd4ef93f6254ee6a7dc8a079f58c6c6 : POST_FAILURE in 1m 23s
- tox-pep8 finger://zuul.v3.opentechsjc.bonnyci.org/3d032f20258946cea1fc6819ebe9f353 : POST_FAILURE in 1m 24s

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

**Comment by jamielennox** [comment]
recheck

File: .zuul.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- project:
+    name: BonnyCI/requests-mock
+    check:
+      jobs:
+        - tox-py27:
+          pre_run:
```
Comment (pabelanger) [inline .zuul.yaml]: pre-run

File: .zuul.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+            - sudo-test
```
Comment (pabelanger) [inline .zuul.yaml]: playbooks/sudo-test

**Pull request closed without merging**